.github/workflows/run-pr-tests.yaml

```
@@ -41,6 +41,14 @@ jobs:
     needs: build-noobaa-image
     uses: ./.github/workflows/warp-nc-tests.yaml
 
+  mint-tests:
+    needs: build-noobaa-image
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 4

(indentation)